polars/polars-core/src/series/mod.rs

```diff
@@ -1038,6 +1047,23 @@ mod test {
         let _ = series.slice(4, 2);
     }
 
+    #[test]
+    fn empty_series_sum_evals_zero() {
```

Comment:
Could these tests moved to the python tests? We want to reduce the testing ci time. Rust tests need a long time to compile/run.